src/components/Sidebar/SidebarCTA.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import { PropsWithChildren, FC } from 'react';
+import { Color } from '../Button';
+
+interface SidebarCTAProps extends PropsWithChildren<Record<string, unknown>> {
```

Comment:
I guess that @bacali95 will say: use type instead interface :)

Reply:
Yes I will say that indeed :slightly_smiling_face: